### DISCUSSION TITLE
Genesis Mini won't turn back on unless...

### DISCUSSION THREAD

**Discussion by TortugaV6SC** [TeamShinkansen/Hakchi2-CE#453] [Q&A]
Hi all,
Genesis Mini will shutdown but not turn back on unless I unplug the power cable and then switch the power switch to "ON" and then plug the power cable back in. 
I tried resetting back to factory, reset kernel, reinstalled kernel, etc. Also, this issue occurs when hakchi is uninstalled too.

Anyone else run into this issue? Possible fix?

**Comment by PCycle2** [comment]
I have a similar issue, but different. The off switch only works if I'm in the main menu or in a Genesis game thats running through the stock emulator. I have to hit the reset button twice to get back to the main menu, then switch it off. I wanted to report the issue somewhere but I'm new to this scene and not sure the right place.
